examples/launcher/src/main.rs

```diff
@@ -0,0 +1,68 @@
+// Launcher used as docker entrypoint for servers.
```

Comment:
So the launcher is basically some kind of CLI that just calls `binary_name server`?
Could this not just be a bash script similar to what you have for the wasm clients?

Reply:
it could be a bash script yeah, except in order to keep the resulting docker image as tiny as possible, so deployment is as fast as possible, it's using an absolute bare-bones docker image with just our rust binaries copied in.
 https://github.com/GoogleContainerTools/distroless 

it doesn't include a shell like bash or sh. there is a version that includes a shell, but that adds more size

normally you'd just set the docker ENTRYPOINT when launching the container, depending on which example, to say "/apps/spaceships/spaceships" and not need a launcher, but that's not exposed via the edgegap dashboard, and setting envs is. if they add entry point settings to the dashboard we can delete the launcher.

Reply:
the bash version is something like this 😄 

```bash
#!/bin/bash
exec /apps/$EXAMPLE_NAME/$EXAMPLE_NAME server
```